base/sysimg.jl

```diff
@@ -400,6 +400,24 @@ end
 INCLUDE_STATE = 3 # include = include_from_node1
 include("precompile.jl")
 
+@noinline function test_clone_f(a)
+    s = zero(eltype(a))
+    @inbounds @simd for i in 1:length(a)
+        s += a[i]
+    end
+    return s
+end
+
+@noinline function test_clone_g(a, n)
+    s = zero(eltype(a))
+    for i in 1:n
+        s += test_clone_f(a)
+    end
+    return s
+end
+
+test_clone_g(Float64[], 1)
```

Comment:
This will be gone in the final version. It's just here for testing. It's pretty hard to actually come up with a useful test case that handles all machines the test can run on....